java-checks/src/main/java/org/sonar/java/symexec/SymbolicEvaluator.java

```diff
@@ -80,13 +82,14 @@
 
   private int currentExecutionStateCount;
 
-  public Map<Tree, SymbolicBooleanConstraint> evaluateMethod(ExecutionState state, MethodTree tree) {
+  public Map<Tree, SymbolicBooleanConstraint> evaluateMethod(JavaFileScannerContext context, ExecutionState state, MethodTree tree) {
     result.clear();
     if (tree.block() != null) {
       try {
         evaluateStatement(ImmutableList.of(state), tree.block());
       } catch (SymbolicExecutionException e) {
-        LOGGER.info("analysis of " + tree + " failed: " + e.getMessage(), e);
+        JavaTree javaTree = (JavaTree) tree;
+        LOGGER.debug("in " + context.getFile() + ": analysis of " + tree.simpleName().name() + " at line " + javaTree.getLine() + " failed: " + e.getMessage(), e);
```

Comment:
I would separate the message that should be logged as info from the exception that should be logged as debug as we don't want to pollute the log with a stacktrace at all times.